src/main/java/com/minecolonies/colony/Colony.java

```diff
@@ -205,6 +228,18 @@ protected void writeToNBT(NBTTagCompound compound)
         }
         compound.setTag(TAG_CITIZENS, citizenTagList);
 
+        //  Achievements
+        final NBTTagList achievementsTagList = new NBTTagList();
+        for (final Achievement achievement : this.colonyAchievements)
+        {
+            // TODO: Validate this
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1135)